ui/helpers/utils/accounts.js

```diff
@@ -60,3 +66,27 @@ export function getAvatarNetworkColor(name) {
       return undefined;
   }
 }
+
```

Comment:
how hard would it be to convert this file to typescript? If its not a major change can we do that here before this file grows too big?

Reply:
Good idea but probably now feasible in this pr